docs/_documentations/codewind-openapi-vscode.md

```diff
@@ -22,6 +22,21 @@ The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI
 3. This extension pulls the [OpenAPI Generator CLI Docker Image](https://github.com/OpenAPITools/openapi-generator#16---docker) and runs the OpenAPI Generator in a Docker container. Install Docker if necessary.
 4. Install the Codewind OpenAPI Tools from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-openapi-tools) or by searching for `Codewind OpenAPI` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 
+## Generating and building code in an existing Java Maven project
+1. Right-click your Codewind project and launch the **Generate Server from OpenAPI Definition** wizard.
+2. Select your OpenAPI document `.json` file.
+3. Select the `jaxrs-spec` server generator type.
+4. When you're prompted to select the output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
+5. Click **Select As Output Folder** to merge the generated `.pom` file with the existing `.pom` file. The merge attempts to include all necessary dependencies for the project to build properly.
+6. When prompted with the warning message, note that the `pom.xml` file is backed up. However, XML comments, such as `<!-- example comment -->`, are removed. Click **Yes** to proceed. The files build.
+7. See that your `src` folder shows the newly generated code complete with a `gen` package. Your project state returns to the `[Running]` state. The `src/main/java` and `src/test/java` are not customizable.
+8. Now, you can customize the application logic.
+
+## Generating and building code in an existing Java Spring project
+When you generate into a Spring project, don't override the implicitly or explicitly configured main class that comes with the project.
+1. To use the generated class, comment out the main class text in the project `.java` file.
+2. To use the current application, copy the packages at the component scan annotation and move them to the `SBApplication.java` file. Then, launch the project.
```

Comment:
2. To continue to use the currently configured main class, copy the base packages listed in the component scan annotation `@ComponentScan` from `OpenAPI2SpringBoot.java` and move them to the currently configured main class. Then, launch the project.   For example, if you are using the Codewind Spring Boot project, the implicit main class is the `SBApplication.java` file.